test/c/test_sparse_observable.c

```diff
@@ -746,6 +828,7 @@ int test_sparse_observable(void) {
     num_failed += RUN_TEST(test_direct_fail);
     num_failed += RUN_TEST(test_obs_str);
     num_failed += RUN_TEST(test_obsterm_str);
+    num_failed += RUN_TEST(test_apply_layout);
```

Comment:
```suggestion
    num_failed += RUN_TEST(test_apply_layout);
    num_failed += RUN_TEST(test_apply_layout_too_small);
```

Reply:
If we get in the habit of declaring our test functions as `static`, I think the linter should warn about it being unused.